ProjectLighthouse/Helpers/ReflectionHelper.cs

```diff
@@ -0,0 +1,25 @@
+﻿#nullable enable
+using System.Reflection;
+
+namespace LBPUnion.ProjectLighthouse.Helpers;
+
+public static class ReflectionHelper
+{
+
+    public static void sanitizeStringsInClass(object? instance)
+    {
+        if (instance == null) return;
+        PropertyInfo[] properties = instance.GetType().GetProperties();
+        foreach (PropertyInfo property in properties)
+        {
+            if (property.PropertyType != typeof(string)) continue;
+
+            string? before = (string?) property.GetValue(instance);
+            if (before == null) continue;
+            if (!before.Contains('>') || !before.Contains('<')) continue;
+            
+            property.SetValue(instance, before.Replace("<", "&lt;").Replace(">", "&gt;"));
```

Comment:
This could be written better for extensibility, say for example other characters can be abused in such a way, or we could even use this for censoring user input.

Here's what I'm thinking:
1. Have a dictionary mapping `char`s to `string`s, populated with the appropriate replacements:
```csharp
private static readonly Dictionary<char, string> charsToReplace = new() {
    {'<', "&lt;"},
    {'>', "&gt;"},
};
```
Probably not the right syntax but the idea is there.

2. Replace the check on line 19 with something like this:
```csharp
if(!charsToReplace.Any(before.Contains)) continue; // Stop if there are no characters that need to be replaced
```

3. Then, replace the characters like so:
```csharp
string after = "";

foreach(KeyValuePair<char, string> entry in charsToReplace)
{
    after = after.Replace(entry.Key, entry.Value);
}

property.SetValue(instance, after);
```